python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -750,17 +784,35 @@ def _throughput_calculate_helper(iterrun, baseline_time, func, *args):
     return np.mean(time_list) * 1000, True
 
 
+def _signature_check(function):
+    '''
+    A quick helper to judge whether input function is following this calling
+    method `metric(pred, target)`.
+    '''
+    import inspect
+    sig = inspect.signature(function)
+    if len(sig.parameters.values()) != 2:
+        return False
+    param1_name = list(sig.parameters.values())[0].name
+    param2_name = list(sig.parameters.values())[0].name
+    if "pred" in param1_name:
+        return True
+    if "target" in param2_name:
+        return True
```

Comment:
`if "pred" in param1_name and "target" in param2_name`, yes?

Reply:
will change to this.